build.sh

```diff
@@ -11,8 +11,8 @@ mkdir -p bin
 rm bin/dataplicity
 virtualenv -qq .build
 source .build/bin/activate
-pip -q install pex==1.1.14
+pip -q install pex==1.2.6
 echo building ./bin/dataplicity
-pex dataplicity -r requirements.txt -o bin/dataplicity -m dataplicity.app:main
+pex dataplicity --pre -r requirements.txt -o bin/dataplicity -m dataplicity.app:main
```

Comment:
The `--pre` allows pex to use alpha builds of libs if specified. This was the previous behaviour.